ha/virsh/vm_utils.sh

```diff
@@ -22,6 +22,11 @@ CLUSTER_NAME="test-cluster"
 HA_NETWORK="ha-${UBUNTU_SERIES::1}-${AGENT}"
 HA_NETWORK=${HA_NETWORK::15}
 
+DHCP_CLIENT="dhclient"
+if [[ "${UBUNTU_SERIES}" = "noble" ]]; then
```

Comment:
This however will break post-noble, right? Should we do the opposite instead? Something like this (untested!)


```
DHCP_CLIENT=dhcpcd
if [[ "${UBUNTU_SERIES}" = "jammy" ]]; then # and maybe mantic? or let's remove it?
    DHCP_CLIENT="dhclient"
fi
```

Reply:
Thanks Paride! +1; I will adjust this (and another place where we are checking for noble) and update this PR.

Reply:
Hey,

When using `[[` (bash-only), the string comparison operator should be `==`.  I think there is some lenience in accepting `=`, but IMHO it's better to stick with `==` here.

Just thought I'd give my 1/2 cent ;).